package/scripts/install-npm.js

```diff
@@ -3,13 +3,20 @@ const path = require("path");
 
 console.log("Updating symlinks for Android build...");
 
+const isExists = (path) => {
+  return fs.existsSync(path);
+}
+
 const createSymlink = (p) => {
   console.log(`Creating symlink to ${p}`, __dirname, process.cwd());
-  fs.symlinkSync(
-    path.resolve(`./cpp/${p}`),
-    path.resolve(`./android/cpp/${p}`),
-    "dir"
-  );
+  const srcDir = path.resolve(`./cpp/${p}`);
+  const dstDir = path.resolve(`./android/cpp/${p}`);
+
+  if (isExists(dstDir)) {
```

Comment:
```suggestion
  if (fs.existsSync(dstDir)) {
```
For me, the _existsSync_ method have a pretty clear name. We can avoid create an another function.